lib/src/isolate.dart

```diff
@@ -318,6 +329,35 @@ class VMIsolateRef {
     }
   }
 
+  Future<VMReloadReport> reloadSources(
+      {bool force, bool pause, rootLibUri, packagesUri}) async {
```

Comment:
Name these "uri" parameters "url" instead. "Uri" is basically a deprecated term at this point.